pkg/kp/service.go

```diff
@@ -26,14 +27,27 @@ func (c consulStore) RegisterService(manifest pods.Manifest, caPath string) erro
 	}
 
 	if manifest.StatusPort != 0 {
+		hostname, err := os.Hostname()
+		if err != nil {
+			return err
+		}
 		podService.Port = manifest.StatusPort
 		podService.Check = &api.AgentServiceCheck{
 			Interval: checkInterval,
 		}
 		if manifest.StatusHTTP {
-			podService.Check.Script = fmt.Sprintf(httpStatusCheck, manifest.StatusPort)
+			podService.Check.Script = fmt.Sprintf(
+				hostname,
```

Comment:
I am very surprised that `hostname` is coming before `httpStatusCheck`! is there some magic I am no aware of to make this work?